### PR TITLE
fix mission_control mention in README

### PR DESCRIPTION
**Pull request by bbonamin**
fix mission_control mention in README

mission_control-jobs has been released, so the README needs some minor updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,8 +229,7 @@ failed_execution.retry # This will re-enqueue the job as if it was enqueued for
 failed_execution.discard # This will delete the job from the system
 ```
 
-We're planning to release a dashboard called _Mission Control_, where, among other things, you'll be able to examine and retry/discard failed jobs, one by one, or in bulk.
-
+We've recently open sourced [mission_control-jobs](https://github.com/basecamp/mission_control-jobs), where, among other things, you're be able to examine and retry/discard failed jobs.
```
Comment (rosa) [inline README.md:232]:
```suggestion
However, we recommend taking a look at [mission_control-jobs](https://github.com/basecamp/mission_control-jobs), a dashboard where, among other things, you can examine and retry/discard failed jobs. 
```

**Review by rosa — CHANGES_REQUESTED** [review]
Thanks @bbonamin! I had a to-do for me to do this and never got around to it, so thank you! I added a minor suggestion to improve readability and fix a typo.

**Comment by bbonamin** [comment]
Thank you too for such a great library!

**Review by rosa — APPROVED** [review]
Thank you! 🙏

**Pull request merged**